.github/workflows/python-app.yml

```
@@ -0,0 +1,37 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
```

Build is a bit generic ... maybe use something like this to provide a hint for what this job is all about.

```suggestion
  ubuntu_2004:
```